improver/snow_fraction.py

```diff
@@ -121,14 +140,17 @@ def _calculate_snow_fraction(self):
                 Snow fraction cube.
 
         """
+        with np.errstate(divide="ignore", invalid="ignore"):
+            snow_fraction = self.snow.data / (self.rain.data + self.snow.data)
```

Comment:
You expect this to produce `np.nan` where there is no precipitation.  How does the `SignificantPhaseMask` plugin handle this?

Reply:
It doesn't have to. The snow-fraction plugin handles these by interpolating to the NaN points from nearby valid points.

Reply:
In which case, is there a test for the snow fraction plugin to ensure it does that please?

Reply:
On discussion - yes there is, it just wasn't immediately clear from the docstring.  Stephen is going to update the docstring to specify that it tests a point with no precip - and the number that comes out is not masked or nan.

Reply:
Done